core/testing.go

```
@@ -24,8 +25,15 @@ import (
 
 // MustGetNonce returns the next nonce for an actor at the given address or panics.
```

this comment should probably be updated. as stated i would expect panic if the actor at the given address is not found

---

I changed it to panic instead 😄